ccpm/scripts/pipeline-lib.sh

```
@@ -0,0 +1,1049 @@
+    return 0
+  fi
+
+  # Check if skipped
+  if [ "$current_status" = "skipped" ]; then
+    echo "⊘ Previously skipped"
+    return 0
+  fi
+
+  # Check if step is skippable
+  local is_skippable=false
+  if [[ " $SKIPPABLE_STEPS " =~ " $step_num " ]]; then
```

Shellcheck (0.11.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)